batches/batch.service.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+
+import Batch from './batch';
+
+class BatchService {
+    private URI: string;
+    constructor() {
+        this.URI = /*process.env.CALIBER_URI*/'https://aosczl5fvf.execute-api.us-west-2.amazonaws.com/default' + '/batches';
+    }
+
+    getBatchesByTrainerEmail(): Promise<Batch[]> {
```

Comment:
would it be better to declare a trainer email outside this function and then pass in the trainer email into this function as an argument?

Reply:
nevermind, the trainer email is not in the store.

Reply:
let trainerEmail = 'mock1027.employee74df14df-5842-4811-a57c-be9836537a40@mock.com'
getBatchesByTrainerEmail(trainerEmail): Promise<Batch[]> {

and then 
return axios.get(this.URI, {params: {trainerEmail: trainerEmail}})